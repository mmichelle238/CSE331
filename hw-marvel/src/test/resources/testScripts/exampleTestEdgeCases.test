# Tests some edge cases with the output format of the scripting language.
# Your MarvelTestDriver implementation should pass this test without you needing
# to change this test.

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddEdge g n1 n2 theOnlyEdge

FindPath g n1 n2

FindPath g n1 n3

FindPath g n1 x

FindPath g x n1

FindPath g x y

FindPath g y x

FindPath g x x

FindPath g n1 n1
