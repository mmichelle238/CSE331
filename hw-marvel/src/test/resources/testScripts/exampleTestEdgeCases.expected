# Tests some edge cases with the output format of the scripting language.
# Your MarvelTestDriver implementation should pass this test without you needing
# to change this test.

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added edge theOnlyEdge from n1 to n2 in g

path from n1 to n2:
n1 to n2 via theOnlyEdge

path from n1 to n3:
no path found

unknown character x

unknown character x

unknown character x
unknown character y

unknown character y
unknown character x

unknown character x
unknown character x

path from n1 to n1:
